src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -72,6 +72,35 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
   const [reviewNote, setReviewNote] = useState<string>(
     pia?.review?.mpo?.reviewNote || '',
   );
+
+  const enableConfirmButton = () => {
+    if (pia.hasAddedPiToDataElements === false) {
+      if (reviewNote === '') return true;
```

Comment:
What if there is  a bunch of spaces:
can we make sure we do this - check if there a re not empty spaces:
str.split(' ').join('')